src/js/player.js

```diff
@@ -502,7 +502,8 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted
+      'muted': this.options_.muted,
+      'poster': this.options_.poster
```

Comment:
@eXon could you change this to use `this.poster()`? That would make sure that if the poster changes, later techs will get the new poster.